MIP0/mip0.md

```diff
@@ -301,153 +263,82 @@ Entries into this list should follow the following template:
 
-Cohesion and Morale
+##### Cohesion and Morale
 - Responsible for raising community governance issues to the Maker Foundation or the third-party ecosystem and ensuring appropriate follow up for the community.
 - Should help to build and maintain morale and engagement among members of the governance community.
-- Should encourage the community to come to consensus over the least objectionable option(s) rather than treating decision-making as a competition where a subset of the community must end up disappointed in the outcome. 
-- Should work to bring the governance community together on divisive topics and to prevent political polarisation and demagoguery. 
-
-**Selection Criteria**
-    
-The following criteria should be used when evaluating an individual for the role of Governance Facilitator:
-
-- Should have a complete understanding of the MIPs Framework and content, especially in relation to core governance MIPs.
-- Required to be a community member for some time. This can be shown through various proof of participation methods, such as:
-	- Past forum posts
-	- Attendance of community and governance calls
-	- Input into issues of governance in any communications venue.
-- Knowledge share will occur when onboarded but the candidate must be familiar with the roles and responsibilities of Governance Facilitators.
-- Should have familiarity with the technical inner workings of the Maker Protocol (bonus)
-- Must have experience engaging with different stakeholders in the community in all the different venues the community uses for communications including chat rooms, forums and video conference calls.
-- Should be confident in expressing themselves in each of the different venues the community uses for communications including chat rooms, forums and video conference calls.
-- Should have an interest in governance mechanisms used presently and historically across the world.
-
-**Removal**
-
-A Governance Facilitator should be considered for removal if they are:
--   Not adequately performing their defined duties
--   Absent from their duties for a prolonged period
--   Displaying biased or malicious behaviour
--   Expressing unwillingness to continue in their role.
-    
-The removal process begins once the community has agreed on the reasoning for removal. This process must be communicated publicly via the forums in order to provide complete transparency. **The Governance Facilitator will then be removed from the following channels:**
--   Github
--   RocketChat
--   Forums
+- Should encourage the community to come to consensus over the least objectionable option(s) rather than treating decision-making as a competition where a subset of the community must end up disappointed in the outcome.
+- Should work to bring the governance community together on divisive topics and to prevent political polarisation and demagoguery.
 
 ---
 
-### MIP0c12: Core Personnel Onboarding
```

Comment:
We should keep these but mark them as obsolete and refer people to the Core Unit MIPs.